common/src/main/java/com/revenuecat/purchases/common/AppConfig.kt

```diff
@@ -18,7 +18,7 @@ class AppConfig(
     var forceServerErrors: Boolean = false
 ) {
 
-    val enableOfflineEntitlements = false
+    val enableOfflineEntitlements = true
```

Comment:
I'm not removing the flag just yet, in case we have to disable it again. Will remove the flag after it's been released.